Emby.Server.Implementations/Networking/NetworkManager.cs

```diff
@@ -268,6 +273,12 @@ public bool IsAddressInSubnets(IPAddress address, bool excludeInterfaces, bool e
             string excludeAddress = "[" + addressString + "]";
             var subnets = LocalSubnetsFn();
 
+            // Include any address if LAN subnets aren't specified
```

Comment:
Re-instates 10.5 functionality. If the LAN subnets are empty - all addresses are deemed to be a local address. (see #3515 for conversation around this).